components/tikv_util/src/sys/cpu_time.rs

```diff
@@ -34,19 +37,57 @@ impl LiunxStyleCpuTime {
             + self.steal
     }
 
-    pub fn current() -> io::Result<LiunxStyleCpuTime> {
+    pub fn current() -> io::Result<LinuxStyleCpuTime> {
         imp::current()
     }
 }
 
+pub use std::io::Result;
+
+pub use imp::cpu_time;
+
+/// A struct to monitor process cpu usage
+#[derive(Clone, Copy)]
+pub struct ProcessStat {
```

Comment:
How about extracting it to a separated pr?

Reply:
It's ok to make a new pr.  But since this function is highly dependent on it. I would like to conclude it in this pr.